### ISSUE TITLE
It is not section 508 compliant

### ISSUE THREAD

**Issue by kbecht** [project-open-data/project-open-data-dashboard#138]
I am color-blind and cannot see the colors in the chart. You should run a 508-checker against it to verify that the other section 508 points have also been addressed.

**Comment by alex-perfilov-reisys** [comment]
@kbecht , thanks for raising the issue. 
Which chart are you referring to?

**Comment by kbecht** [comment]
I’m sorry, but I’m not a regular GitHub user.  The chart I was referring to is at http://labs.data.gov/dashboard/offices.

Kimberly J. C. Becht
Kimberly.Becht@trade.gov<mailto:Kimberly.Becht@trade.gov>
Lead IT Specialist, Requirements Elicitation
Office of the Chief Information Officer
U.S. Department of Commerce, International Trade Administration
Office: (202) 482-0420
Blackberry: (202) 295-7680

**Comment by alex-perfilov-reisys** [comment]
@kbecht , thanks. Actually, the colors are not so important when icons work fine. Looks like on http the icons are not displayed. Can you try `https://` version ? https://labs.data.gov/dashboard/offices
It has helpful icons which show the status even without color

<img width="495" alt="screenshot 2016-04-19 14 36 01" src="https://cloud.githubusercontent.com/assets/6089683/14651041/33e26758-063c-11e6-9e36-2977aff13ec5.png">

we will address the issue, of course

**Comment by kbecht** [comment]
I’m still not seeing any icons:

[cid:image001.png@01D19A4C.E6BAF860]

Kimberly J. C. Becht
Kimberly.Becht@trade.gov<mailto:Kimberly.Becht@trade.gov>
Lead IT Specialist, Requirements Elicitation
Office of the Chief Information Officer
U.S. Department of Commerce, International Trade Administration
Office: (202) 482-0420
Blackberry: (202) 295-7680

**Comment by alex-perfilov-reisys** [comment]
@kbecht , we are not able to see your image. Please use the link below to answer on GitHub website. Just drag&drop the image and click 'comment' . Make sure you're using https:// version. If you can provide your browser information, that would be helpful, too. Thanks

**Comment by kbecht** [comment]
![dashboard capture](https://cloud.githubusercontent.com/assets/18551375/14708736/b82cdd04-079a-11e6-902e-e53567990078.JPG) I am using IE version 11.0.9600.18231 update version: 11.0.29(KB3139929)

**Comment by justgrimes** [comment]
@kbecht Thanks for bringing this issue up. The issue you are having is the same as #70 & #113; I believe there is a security setting in IE that is preventing fonts from being downloaded. For the logos to display properly you'll need to be able to load these fonts. If you are still having this problem either add this site as a trusted site in Internet Explorer or from Internet Explorer go in to "Options" then "Security" then "Internet zone", "Downloads" then make sure "Font Download" is set to Enable. 

I'm going to leave this issue open to continue the discussion for alternatives to displaying this information that are not color dependent.

**Comment by QJF3** [comment]
At least for many of us in the VA, all the options that @justgrimes described for IE are disabled and require a desktop administrator to make any changes.  Any suggestions how to view the page icons given these constraints?

John Quinn
US Department of Veterans Affairs
